Gemfile

```diff
@@ -55,11 +53,10 @@ gem "sassc-rails"
 gem "pundit"
 gem "faker"
 gem "open-uri"
-gem "nokogiri"
-gem "watir"
+gem "geocoder"
 gem "pg_search"
 gem "ruby-openai"
-gem "redis"
```

Comment:
why is redis being removed?

Reply:
Not being used for caching, but I can re-add if it breaks the app.